lib/ansible/module_utils/facts/hardware/linux.py

```diff
@@ -443,25 +443,13 @@ def _mtab_entries(self):
             mtab_entries.append(fields)
         return mtab_entries
 
-    @timeout.timeout()
-    def get_mount_facts(self):
-        mount_facts = {}
-
-        mount_facts['mounts'] = []
+    def get_mount_info(self, fields, bind_mounts, uuids):
 
-        bind_mounts = self._find_bind_mounts()
-        uuids = self._lsblk_uuid()
-        mtab_entries = self._mtab_entries()
-
-        mounts = []
-        for fields in mtab_entries:
             device, mount, fstype, options = fields[0], fields[1], fields[2], fields[3]
+            mount_info = {}
 
-            if not device.startswith('/') and ':/' not in device:
-                continue
-
-            if fstype == 'none':
-                continue
+            if all([not device.startswith('/'), ':/' not in device, not fstype.startswith('fuse')]) or fstype == 'none':
```

Comment:
It's more readable to write this out into multiple if-then statements.

Reply:
moved and reverted